### PR TITLE
Align form fields

### LINKED ISSUE #1880: UX: Align all the form fields to end at the same place
Try to find a way to have all form fields both start and end at the same place, regardless of nesting level.

This includes the alignement of buttons and key value pairs. 

image

### PR DESCRIPTION
**Pull request by dariadomagala**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

Resolves #1880

### CODE REVIEW

File: core-ui/src/shared/components/Select/Select.js [hunk 1]
```diff
@@ -9,5 +9,9 @@ export function Select({ fullWidth, ...props }) {
     };
   }
 
-  return <FdSelect {...props} />;
+  return (
+    <div className="fd-col fd-col-md--11">
+      <FdSelect {...props} />{' '}
```
Comment (Wawrzyn321) [inline core-ui/src/shared/components/Select/Select.js:14]:
```suggestion
      <FdSelect {...props} />
```

**Review by Wawrzyn321 — CHANGES_REQUESTED** [review]
Would you kindly fix the misaligned preferences?
image
image

2nd one has a too big of a padding.

File: core-ui/src/shared/ResourceForm/components/CollapsibleSection.scss [hunk 2]
```diff
@@ -28,7 +28,7 @@
 
   .content {
     padding: {
-      left: 16px;
+      // left: 16px;
```
Comment (Wawrzyn321) [inline core-ui/src/shared/ResourceForm/components/CollapsibleSection.scss:31]:
```suggestion
```
Or just revert entire file.

File: core-ui/src/shared/ResourceForm/fields/K8sNameField.js [hunk 3]
```diff
@@ -44,7 +44,7 @@ export function K8sNameField({
       input={() => {
         return (
           <>
-            <div className="k8s-name-field">
+            <div className="fd-col fd-col-md--11">
```
Comment (Wawrzyn321) [inline core-ui/src/shared/ResourceForm/fields/K8sNameField.js:47]: `k8s-name-field` class rule can be removed then.

File: core-ui/src/shared/ResourceForm/fields/TextArrayInput.scss [hunk 4]
```diff
@@ -1,23 +1,23 @@
-.text-array-input__list li {
-  display: flex;
+// .text-array-input__list li {
```
Comment (Wawrzyn321) [inline core-ui/src/shared/ResourceForm/fields/TextArrayInput.scss:1]: To remove?

**Review by Wawrzyn321 — COMMENTED** [review]
Should we align KeyValueFields in BTP stuff?
From:
image

into:
image

So that they look the same as Labels / Annotations. Removing the `fullWidth` prop on their create forms will do.

**Comment by Wawrzyn321** [comment]
Let's keep all change requests here:
image

**Comment by Wawrzyn321** [comment]
Also please align the "type" input:
image

**Comment by Wawrzyn321** [comment]
image

**Comment by Wawrzyn321** [comment]
image

**Comment by dariadomagala** [comment]
> Also the "language" option is misplaced in CMs: image

It was always placed like this

**Review by Wawrzyn321 — APPROVED** [review]

**Review by grego952 — APPROVED** [review]

**Review by Wawrzyn321 — APPROVED** [review]

**Review by grego952 — APPROVED** [review]

**Pull request merged**